lib/ai/chat.rb

```diff
@@ -225,22 +228,64 @@ def create_response
       messages_to_send = prepare_messages_for_api
       parameters[:input] = strip_responses(messages_to_send) unless messages_to_send.empty?
 
-      client.responses.create(**parameters)
+      if proxy
+        uri = URI(PROXY_URL + "api.openai.com/v1/responses")
+        Net::HTTP.start(uri.host, uri.port, use_ssl: uri.scheme == "https") do |http|
```

Comment:
```suggestion
        Net::HTTP.start(uri.host, uri.port, use_ssl: true) do |http|
```

Do we want to allow `http`? Would that ever happen with the `PROXY_URL`? I don't think so, since that is a frozen constant with `https` above.

Also, I think `use_ssl` defaults to `true` if you pass in an `https` URL ([source](https://ruby-doc.org/3.2.2/stdlibs/net/http/Net/HTTP.html#class-Net::HTTP-label-HTTPS))